src/coredump/coredumpctl.c

```diff
@@ -965,7 +965,9 @@ static int dump_list(int argc, char **argv, void *userdata) {
                 if (!arg_field && n_found <= 0) {
                         if (!arg_quiet)
                                 log_notice("No coredumps found.");
-                        return -ESRCH;
```

Comment:
Can we have a workaround in the script instead, at least for 257? That way we can already merge this, and change this return value in 258 and remove the workaround